cephalopod/cephalopod.ini

```diff
@@ -0,0 +1,11 @@
+[uwsgi]
```

Comment:
I thought you meant the repo with the openshift stuff. I don't think it's very useful here. (and certainly not with a name that does not indicate it's a uwsgi config)